kyuubi-common/src/main/scala/org/apache/kyuubi/service/BackendService.scala

```diff
@@ -91,7 +91,7 @@ trait BackendService {
       foreignTable: String): OperationHandle
   def getQueryId(operationHandle: OperationHandle): String
 
-  def getOperationStatus(operationHandle: OperationHandle): OperationStatus
+  def getOperationStatus(operationHandle: OperationHandle, maxWait: Long = 0): OperationStatus
```

Comment:
can we preserve the old method ? we can have both of these:
```
def getOperationStatus(operationHandle: OperationHandle): OperationStatus
def getOperationStatus(operationHandle: OperationHandle, waitTimeMs: Long): OperationStatus
```

Reply:
`Backendservice` mixed with `BackendServiceMetric`, using multiple inheritance to implement the metric getting wrapper, must have the same parent class. Adding a method will cause a `BackenServiceMetric` compilation error

Reply:
I see. Can we remove the default value 0 ? It may cause potential issue.
`def getOperationStatus(operationHandle: OperationHandle, maxWaitMs: Long): OperationStatus`